.github/workflows/verify-data.yml

```diff
@@ -63,14 +63,14 @@ jobs:
         run: exit 1
         if: steps.linkedin_count_friends.outputs.COUNT != '0'
 
-      - name: Find 'Bluesky' in postgresfriends.json file
-        run: echo "COUNT=$(grep -c 'Bluesky' postgresfriends.json)" >>$GITHUB_OUTPUT
+      - name: Find 'BlueSky' in postgresfriends.json file
+        run: echo "COUNT=$(grep -c 'BlueSky' postgresfriends.json)" >>$GITHUB_OUTPUT
```

Comment:
Eh, https://bsky.social/about/support/tos

It's "Bluesky" all over their website.